module/store.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable linebreak-style */
+export default class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(author) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
```

Comment:
- After removing books, they're not correctly saved to the local storage.

![not_saved](https://user-images.githubusercontent.com/27709832/204577021-376c2a13-099a-4c8b-aad2-1ba14ae2b750.gif)

Reply:
Fixed this issue at this [line](https://github.com/Lucash2022/Awesome_Book/blob/009eca8b70ea7e8797c2354c5c6d960e4b5594b4/main.js#L19)